shared-helpers/src/auth/UseRequireLoggedInUser.ts

```diff
@@ -1,16 +1,16 @@
 import { useContext } from "react"
 import { AuthContext } from "./AuthContext"
-import { NavigationContext } from "@bloom-housing/ui-components"
+import { useRouter } from "next/router"
 
 /**
  * Require a logged in user. Waits on initial load, then initiates a redirect to `redirectPath` if user is not
  * logged in.
  */
 function useRequireLoggedInUser(redirectPath: string) {
   const { profile, initialStateLoaded } = useContext(AuthContext)
-  const { router } = useContext(NavigationContext)
+  const router = useRouter()
```

Comment:
Why was this change made?

Reply:
I believe it was because the next router is what we have been using for redirects. For example in forgot-password.tsx or confirmation.tsx. And I don't this hook was utilized anywhere else in the webapp. Do you the difference between using the next router and NavigationContext router? I noticed there are more available function calls on the next router, but wasn't sure why different routers were used in different places.